snap/gui/heroic.desktop

```diff
@@ -0,0 +1,11 @@
+[Desktop Entry]
```

Comment:
could you update this desktop file to match the flatpak one (plus whatever changes the snap one needs ofc)